spec/request/api/v0/ping_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Api::V0::PingController, type: :request do
+  describe 'GET /v0/ping' do
+    it 'should return Pong' do
+      get '/api/v0/ping'
+
+      json_response = JSON.parse(response.body)
+      expect(response.status).to eq 200
+      expect(json_response['message']).to eq 'Pong'
```

Comment:
You have a `response_json` helper. Why not use it? 

But I would get rid of this file now that you know how to set up an API.